src/themes/clean/templates/elements/journal/issue_sidebar.html

```diff
@@ -6,8 +6,8 @@
             <h2>{% trans 'Downloads' %}</h2>
             <p>
                 <a href="{% url 'journal_issue_download_galley' issue.pk issue.galley.pk %}">
-                    <i aria-hidden="true" class="fa fa-download"></i>&nbsp
-                    {% trans 'Download Issue' %} (PDF)
+                    {% trans 'Issue' %} (PDF)
+                    {% include "elements/icons/link_download.html" %}
```

Comment:
Isn't "Download Issue" a better title here? Even if redundant with the icon, it gives a more clear prompt to translators of what the text means.